ConsolePort/Model/Gamepad/Nintendo/N64.lua

```diff
@@ -0,0 +1,166 @@
+select(2, ...).Gamepad:AddGamepad({
+	Name = 'Nintendo N64';
+	LabelStyle = 'N64';
+	Version = 1;
+	Theme = {
+		Label = 'N64';
+		Colors = {
+			PAD1	  = '00A2FF';
+			PAD2	  = '00FF5D';
+		};
+		Icons = {
+			PADDUP       = 'N64/Up';
+			PADDRIGHT    = 'N64/Right';
+			PADDDOWN     = 'N64/Down';
+			PADDLEFT     = 'N64/Left';
+			PAD1         = 'N64/A';
+			PAD2         = 'N64/B';
+			PAD3         = 'N64/CLeft';
+			PAD4         = 'N64/CUp';
+			PAD5         = 'N64/Snapshot';
+			--PAD6
+			PADLSTICK    = 'N64/ZR';
+			--PADRSTICK
+			PADLSHOULDER = 'N64/L';
+			PADRSHOULDER = 'N64/R';
+			PADLTRIGGER  = 'N64/ZL';
+			PADRTRIGGER  = 'N64/CDown';
+			--PADLSTICKUP
+			--PADLSTICKRIGHT
+			--PADLSTICKDOWN
+			--PADLSTICKLEFT
+			--PADRSTICKUP
+			--PADRSTICKRIGHT
+			--PADRSTICKDOWN
+			--PADRSTICKLEFT
+			--PADPADDLE1
+			--PADPADDLE2
+			--PADPADDLE3
+			--PADPADDLE4
+			PADFORWARD  = 'N64/Start';
+			PADBACK     = 'N64/CRight';
+			PADSYSTEM   = 'N64/Home';
+			--PADSOCIAL
+		};
+		Layout = {
+			PADLTRIGGER  = 0x10;
+			PADLSHOULDER = 0x11;
+			--------------------
+			PADDUP		 = 0x12;
+			PADDLEFT	 = 0x13;
+			PADDDOWN	 = 0x14;
+			PADDRIGHT	 = 0x15;
+			--------------------
+			PADLSTICK    = 0x16;
+			PADBACK      = 0x17;
+			PADSYSTEM    = 0x18;
+			--------------------
+			PADRTRIGGER	 = 0x19;
+			PADRSHOULDER = 0x20;
+			--------------------
+			PAD1         = 0x21;
+			PAD2         = 0x22;
+			PAD3         = 0x23;
+			PAD4		 = 0x24;
+			PAD5		 = 0x25;
+			--------------------
+			PADFORWARD   = 0x26;
+			--------------------
+		};
+	};
+	Preset = {
+		Variables = {
+			synchronizeSettings     = 0;

```

Comment:
Synchronization settings should probably not be part of this preset. The reason Steam Deck has those is that it’s an entirely separate device, and you probably don’t want it synchronized with your computer